.github/workflows/dapps_reusable.yml

```diff
@@ -0,0 +1,602 @@
+name: dApps testing
+run-name: dApps testing (${{ inputs.dapps || 'all dapps'}}, ${{ inputs.network }})
```

Comment:
Lets rename 
now there are 2 the same names
My suggesion
1)dApps tests
2)dApps jobs for external stand

Reply:
Agreed to use dApps reusable, as dApps tests also use dApps jobs for external stand inside.